### ISSUE TITLE
Improve `run_blastp_parallel.sh`

### ISSUE THREAD

**Issue by jmtsuji** [jmtsuji/basic-sequence-analysis#2]
Before `run_blastp_parallel.sh` is ready to be merged into master:

- [ ] Add flag-based arguments
- [ ] As default, rely on BLAST itself to find the user's .BLASTB file to direct to the BLAST database, and have an optional flag to specify the database location manually
- [ ] Allow to toggle between BLASTN and BLASTP (and others? make completely generic?)
- [ ] Have the dib lab tool auto-download if needed? (Optional)

**Comment by jmtsuji** [comment]
I no longer think this script is necessary, because BLASTP can be run in parallel using just `blastp` if a DB is made. I will no longer actively develop this code and have moved it to a stale branch, `parallel-blast`. (It was never merged to master.)

**Comment by jmtsuji** [comment]
Note that the code to convert tax IDs to NCBI taxonomy could still be useful in the future, however.
